Source/SVWidgetsLib/FilterParameterWidgets/FilterParameterWidgetUtils.hpp

```diff
@@ -216,7 +216,19 @@ class FilterParameterWidgetUtils
       }
     }
 
-
+    // -----------------------------------------------------------------------------
+    template <typename T>
+    static bool VectorContains(const std::vector<std::vector<T>>& container, const std::vector<T>& comparison)
+    {
+      for(const auto& value : container)
```

Comment:
return std::find(...) != container.end()

Reply:
Didn't know about that one... I thought it was strange there was no way to "find" or "contains" something.

Reply:
Anything with STL iterator support can be used in std::find, std::find_if, and many other interesting functions in `<algorithm>`.  The reason the type doesn't have it is because the method is already generically available.